docs/authentication.md

```diff
@@ -130,6 +130,17 @@ Both of these libraries support either authentication pattern. If you're interes
 - [with-passport](https://github.com/vercel/next.js/tree/canary/examples/with-passport)
 - [with-passport-and-next-connect](https://github.com/vercel/next.js/tree/canary/examples/with-passport-and-next-connect)
 
+### SuperTokens
+
+<details open>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/with-supertokens">with-supertokens</a></li>
+  </ul>
+</details>
+
+[SuperTokens](https://supertokens.io) is an open source secure authentication system with many features such as credentials login, email verification, reset password flows, and third party logins.
```

Comment:
Maybe in your one-sentence summary talk about why you'd chose it over the others? Open source is probably important. "secure authentication system" is likely implied.

Reply:
Good advice! I've updated to reflect that SuperTokens takes a modular approach while being customisable.